### ISSUE TITLE
Pumpjack not working (Immersive Petroleum)

### ISSUE THREAD

**Issue by RyanBerge** [NillerMedDild/Enigmatica2Expert#119]
I'm in a multiplayer server, version 1.30.  I build the multiblock structure and provide power; the pumpjack functions for a moment, pumps about 400mb of crude oil out of the ground, then stops.  I can break the structure, reform it, and repeat the behavior, but I can't get it to just run continuously.

The internal reserve of 16000 IF fills up when I attach the power.  In the moment it takes to fill, it works, then it stops working when the internal power reserve is full.

This is my first time playing with Immersive Petroleum, but I don't think I'm doing anything different than the videos I've seen... =/

**Comment by psykoe** [comment]
Do you have pipes on it pulling out the oil into something (like a distillation tower or tank? Could be just filling it's internal fluid tank and stopping

**Comment by RyanBerge** [comment]
Yeah, it goes into my tank.  About 400mb every time I reform the multiblock.

**Comment by psykoe** [comment]
Very strange. I set up one a few days ago and have both outputs hooked to a distillation tower and it ran for quite some time
